### ISSUE TITLE
Overlay object

### ISSUE THREAD

**Issue by sbrisson** [JuliaGraphics/Gtk.jl#265]
I keep getting the same error when I try to create an Overlay object.

```
using Gtk.ShortNames
overlay = @Overlay()
```

LoadError: UndefVarError: title not defined
 in Gtk.GtkOverlayLeaf() at C:\Users\Saylor Brisson.julia\v0.5\Gtk\src\layout.jl:252
 in include_string(::String, ::String) at .\loading.jl:441
 in include_string(::Module, ::String, ::String) at C:\Users\Saylor Brisson.julia\v0.5\CodeTools\src\eval.jl:32
 in (::Atom.##54#57{String,String})() at C:\Users\Saylor Brisson.julia\v0.5\Atom\src\eval.jl:79
 in withpath(::Atom.##54#57{String,String}, ::String) at C:\Users\Saylor Brisson.julia\v0.5\CodeTools\src\utils.jl:30
 in withpath(::Function, ::String) at C:\Users\Saylor Brisson.julia\v0.5\Atom\src\eval.jl:45
 in macro expansion at C:\Users\Saylor Brisson.julia\v0.5\Atom\src\eval.jl:77 [inlined]
 in (::Atom.##53#56{String,String})() at .\task.jl:60
while loading untitled, in expression starting on line 2 

The source code it points me to in Gtk is, with the error occurring on the bolded line:

if gtk_version == 3
    GtkOverlayLeaf() = GtkOverlayLeaf(ccall((:gtk_overlay_new, libgtk), Ptr{GObject},
        (Ptr{UInt8},), bytestring(title)))
    GtkOverlayLeaf(w::GtkWidget) = invoke(push!, (GtkContainer,), GtkOverlayLeaf(), w)
    function push!(w::GtkOverlay, x::GtkWidget)
        ccall((:gtk_overlay_add_overlay,libgtk), Cint,
            (Ptr{GObject}, Ptr{GObject}), w, x)
    end
end

I try to define title by creating an Overlay like this:

```
using Gtk.ShortNames
overlay = @Overlay("some title")
```

but still get an error.

LoadError: MethodError(Gtk.GtkOverlayLeaf,("some title",))
 in #GtkOverlayLeaf#113(::Array{Any,1}, ::Type{T}, ::String, ::Vararg{String,N}) at C:\Users\Saylor Brisson.julia\v0.5\Gtk\src\GLib\gtype.jl:212
 in Gtk.GtkOverlayLeaf(::String) at C:\Users\Saylor Brisson.julia\v0.5\Gtk\src\GLib\gtype.jl:211
 in include_string(::String, ::String) at .\loading.jl:441
 in include_string(::Module, ::String, ::String) at C:\Users\Saylor Brisson.julia\v0.5\CodeTools\src\eval.jl:32
 in (::Atom.##54#57{String,String})() at C:\Users\Saylor Brisson.julia\v0.5\Atom\src\eval.jl:79
 in withpath(::Atom.##54#57{String,String}, ::String) at C:\Users\Saylor Brisson.julia\v0.5\CodeTools\src\utils.jl:30
 in withpath(::Function, ::String) at C:\Users\Saylor Brisson.julia\v0.5\Atom\src\eval.jl:45
 in macro expansion at C:\Users\Saylor Brisson.julia\v0.5\Atom\src\eval.jl:77 [inlined]
 in (::Atom.##53#56{String,String})() at .\task.jl:60
while loading untitled, in expression starting on line 2

**Comment by tknopp** [comment]
Hi @sbrisson,

yes the implementation in Gtk.jl is wrong. Thanks for reporting the issue. The function `gtk-overlay-new` does not get any arguments:

https://developer.gnome.org/gtk3/stable/GtkOverlay.html#gtk-overlay-new

Would you be interested making a pull request that fixes the issues?

**Comment by jla497** [comment]
Hi, I made the following changes in the layout.jl and GtkOverlayLeaf() works now on my machine. Can you check if the following fix is valid:

if gtk_version == 3
    GtkOverlayLeaf() = GtkOverlayLeaf(ccall((:gtk_overlay_new, libgtk), Ptr{GObject},
      ()))
    GtkOverlayLeaf(w::GtkWidget) = invoke(push!, (GtkContainer,), GtkOverlayLeaf(), w)
    function push!(w::GtkOverlay, x::GtkWidget)
        ccall((:gtk_overlay_add_overlay,libgtk), Cint,
            (Ptr{GObject}, Ptr{GObject}), w, x)
    end
end

**Comment by tknopp** [comment]
was fixed by #266
